_docker/drupal/drupal-filesystem/composer.json

```diff
@@ -18,60 +18,24 @@
         }
     },
     "require": {
-        "acquia/lightning": "2.0.6",
-        "composer/installers": "^1.0",
-        "cweagans/composer-patches": "^1.6.0",
-        "drupal-composer/drupal-scaffold": "2.2.0",
+        "acquia/lightning": "2.1.3",
         "drupal/admin_toolbar": "1.18.0",
-        "drupal/config_update": "1.3.0",
         "drupal/conflict": "1.0.0-alpha1",
-        "drupal/contact_storage": "1.0.0-beta8",
-        "drupal/core": "8.2.7",
-        "drupal/ctools": "3.0.0-alpha27",
-        "drupal/ctools_block": "3.0.0-alpha27",
-        "drupal/diff": "1.0.0-rc1",
+        "drupal/core": "~8.3.1",
```

Comment:
Again I think we should lock the version of all dependencies mentioned in composer.json, but particularly Drupal core.